compiler/damlc/tests/daml-test-files/InterfaceConsuming.daml

```diff
@@ -0,0 +1,43 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- @SINCE-LF-FEATURE DAML_INTERFACE
+-- @ERROR Choice implementation and interface definition for Transfer differ in consuming/non-consuming behaviour.
```

Comment:
maybe add a successful case as well and add a range to this one to make sure that it works if it’s correct and fails if it isn’t.

Reply:
the range will be the beginning of the file (since it's the daml-lf typechecker complaining). There is a successful case in Interface.daml with the `Noop` choice, there it's also tested in the scenario.